tox.ini

```diff
@@ -96,9 +98,8 @@ extras =
 
 commands =
     pip freeze
-
     pytest --pyargs photutils {toxinidir}/docs \
-    cov: --cov photutils --cov-config={toxinidir}/setup.cfg --cov-report xml:{toxinidir}/coverage.xml \
+    cov: --cov photutils --cov-config={toxinidir}/pyproject.toml --cov-report coverage.xml --cov-report term-missing \
```

Comment:
This is causing the CI test to fail:  https://github.com/astropy/photutils/actions/runs/6161981964/job/16723173426?pr=1488#step:6:97

Reply:
I think the `{toxinidir}` path might also be needed here.  We'll see what CI says.

Reply:
Indeed, the coverage file is not found:
https://github.com/astropy/photutils/actions/runs/6162499225/job/16724233090?pr=1488#step:7:70